app/models/user.rb

```diff
@@ -3,5 +3,26 @@ class User < ActiveRecord::Base
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :trackable, :validatable
+         :recoverable, :rememberable, :trackable, :validatable,
+         :omniauthable
+ def self.connect_to_linkedin(auth, signed_in_resource=nil)
```

Comment:
Unused method argument - `signed_in_resource`. If it's necessary, use `_` or `_signed_in_resource` as an argument name to indicate that it won't be used.<br>Inconsistent indentation detected.<br>Surrounding space missing in default value assignment.